bdk-ffi/Cargo.toml

```diff
@@ -25,6 +25,7 @@ bdk_file_store = { version = "0.10.0" }
 uniffi = { version = "=0.26.1" }
 bitcoin-internals = { version = "0.2.0", features = ["alloc"] }
 thiserror = "1.0.58"
+serde_json = "1.0.116"
```

Comment:
I think instead of adding `serde_json` directly here we can add `bdk_bitcoind_rpc = { version = "0.10.0" }` a few lines above.

Then in `bitcoin.rs` we can `use bdk_bitcoind_rpc::bitcoincore_rpc::jsonrpc::serde_json;` and nothing else needs to change (at least from me testing locally).

How does that sound to you?

Reply:
Yeah that's what we had before I think. I wasn't sure if it was cleaner to import the dependency directly or add the bitcoind_rpc lib to our dependencies, but I think because we'll be adding the RPC client anyway this will become a dependency in the future, so it makes sense to just grab it from there.

Reply:
Cool, that makes sense mentioning how we will be adding the rpc client too, good additional context/point